### ISSUE TITLE
Please separate "Windows" as a proper noun from a plural form one

### ISSUE THREAD

**Issue by ryonakano** [elementary/website#1748]
## Actual behavior
In Japanese, the former represents an operating system which Microsoft made.
And the latter represents just a plural form of window, and "W" become capital character because it locales in the top of an sentence.
So these words represent different two things, and have to distinguish.
However, I can't translate separated them because Weblate recognise these words as a same words.

## Expected behavior
The former should be translated as "Windows".
And the latter should be translated as "ウィンドウ".
In other words;
The former, "/_lang/ja/docs/installation.json" of 14th line should be like this;
`"Windows": "Windows",`
The latter, "_lang/ja/docs/learning-the-basics.json" of 33th line should be like this;
`"Windows": "ウィンドウ",`

## Steps to reproduce the behavior
Check these two words in GitHub;
The former, " website/_lang/ja/docs/installation.json" of 14th line is like this:
`"Windows": "ウィンドウ",`
The latter, "_lang/ja/docs/learning-the-basics.json" of 33th line is like this;
`"Windows": "ウィンドウ",`

**Comment by lewisgoddard** [comment]
Ah, that's an interesting problem. Any "fix" for this will likely result in hundreds or thousands of new duplicates from titles.

**Comment by ryonakano** [comment]
I found it possible to distinguish these two words in Weblate: "(Microsoft) Windows" and "(Some) Windows," although it was impossible when I posted this issue. I guess the update of Weblate enabled it.
I appreciate it.
